articles/iot-edge/how-to-store-data-blob.md

```diff
@@ -190,7 +190,7 @@ The following quickstart samples use languages that are also supported by IoT Ed
 
 ## Connect to your local storage with Azure Storage Explorer
 
-You can use [Azure Storage Explorer](https://github.com/microsoft/AzureStorageExplorer/releases/tag/v1.14.2) to connect to your local storage account.
+You can use [Azure Storage Explorer] (https://azure.microsoft.com/features/storage-explorer/) to connect to your local storage account.
```

Comment:
@arduppal There's a space between the ] and ( which will break the link from rendering. Please remove that, then I can sign off.

Reply:
done